source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveCardRenderer.cs

```diff
@@ -228,7 +228,8 @@ void MediaClickCallback(object sender, AdaptiveMediaEventArgs args)
                 Config = HostConfig ?? new AdaptiveHostConfig(),
                 Resources = Resources,
                 ElementRenderers = ElementRenderers,
-                Lang = card.Lang
+                Lang = card.Lang,
+                ForegroundColors = (HostConfig == null) ? HostConfig.ContainerStyles.Default.ForegroundColors : new ContainerStylesConfig().Default.ForegroundColors
```

Comment:
>(HostConfig == null) ? HostConfig.ContainerStyles.Default.ForegroundColors [](start = 35, length = 74)

is this backwards?

Reply:
(and below)

---
In reply to: [257000986](https://github.com/Microsoft/AdaptiveCards/pull/2395#discussion_r257000986) [](ancestors = 257000986)